src/components/scenes/NotificationScene.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import { type EdgeCurrencyInfo } from 'edge-core-js'
-import * as React from 'react'
+import React, { Component } from 'react'
```

Comment:
We just did a giant refactor to switch everything over to `import * as React from 'react'`, so please leave this the way it was. The * import is the "correct" way to use React going forward, according to tweets from the React team. Also, it's the only way that works out-of-the-box in Typescript without special config tweaks.